doc/rpc/components/schemas/BlockExampleResultFullTransaction.json

```diff
@@ -0,0 +1,39 @@
+{
+  "BlockExampleResultFullTransaction": {
+    "name": "block",
+    "value": {
+      "number": "0x1b4",
+      "hash": "0xf1652d8322a880e520f996f7d28b645814a58a202d7d2ab7f058e5566fe4f9f3",
+      "parentHash": "0x9203a97b2595a44e8d31dd7bf607155620dd50982ac282fc776e21a58b2f7795",
+      "sha3Uncles": "0x2f752c2e72d38f3677bfa47bd54ec10441141a6415f3be121341d94276dfd732",
+      "logsBloom": "0x00000008000000800000000000000000000000000000000000000000000008000000000000040000000000000000000050000000000000000000000000000000000000000000000000000000005000000010000000000000100000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000200000000000200000000000001040000000000000400000000000000000000100000000000000010000000000000000000001000000000000001000001000000000000000000000000000020000000000080200000100000000000000000000000000000000000000000000000000000000000000000000000",
+      "transactionsRoot": "0xe94962ab91fe305207670ca17d958ce15bdd553aa6630eba07476c0f35a65b50",
+      "stateRoot": "0xa12e5b995ec8056d6c2d2558c6f4e1ab7f0d6d3eaed868e84c00923cf58bb642",
+      "receiptsRoot": "0x2cdc6fdc519baad55c4f9d86d1415dbfe21265c6b2ba1f283e1391e015735c42",
+      "miner": "0x1fab9a0e24ffc209b01faa5a61ad4366982d0b7f",
+      "difficulty": "0x1aa2b738",
+      "totalDifficulty": "0x1f61daa8d6",
+      "extraData": "0x",
+      "size": "0xafe",
+      "gasLimit": "0x67c280",
+      "gasUsed": "0x0",
+      "timestamp": "0x5d1f7c06",
+      "transactions": [
+        {
+          "$ref": "#/components/schemas/TransactionExampleResult"
+        }
+      ],
```

Comment:
... and following on from prev comment, this should have 1 `TransactionRemascExampleResult` followed by a `TransactionExampleResult`

Reply:
Sure, as soon as the task related to the Remasc fix gets implemented (it's WIP now), its related OpenRPC doc will be updated accordingly.